dcc-release-core/src/main/java/org/icgc/dcc/release/core/util/DocumentRDDs.java

```diff
@@ -46,6 +54,29 @@
         .map(tuple -> new Document(type, tuple._1, tuple._2));
   }
 
+  public static void saveAsTextObjectNodeFile(@NonNull JavaRDD<Document> rdd, @NonNull String path) {
+    val output = rdd.map(row -> MAPPER.writeValueAsString(row.getSource()));
```

Comment:
That's a `String`, not `JSON`. I think that's the only place when this method is used with this particular mapper(because there're mapper with other settings are created in the project).